tests/operator_test.go

```diff
@@ -1021,7 +1021,7 @@ spec:
 		// running a VM/VMI using that previous release
 		// Updating KubeVirt to the target tested code
 		// Ensuring VM/VMI is still operational after the update from previous release.
-		It("[owner:@sig-compute][test_id:3145]from previous release to target tested release", func() {
+		It("[QUARANTINE][owner:@sig-compute][test_id:3145]from previous release to target tested release", func() {
```

Comment:
this is the update test. we shouldn't quarantine it. It's one of the few release blocker tests I identified in this pr, #5182 

if it's still failing, that needs to be investigated asap. I'll continue to look into this.

Reply:
i'm looking into this test a little further here #5194 

For pr #5182 (the release-blocker marker pr) can you think of a way we can enforce that these tests aren't removed from rotation? There's only a few truly critical tests

Reply:
> i'm looking into this test a little further here #5194

Awesome thanks a lot!

> For pr #5182 (the release-blocker marker pr) can you think of a way we can enforce that these tests aren't removed from rotation? There's only a few truly critical tests

For sure, now we are tagging the tests manually, hopefully we'll have an automated procedure soon. In both cases, if the `[]release-blocker` label is found we can prevent adding the quarantine label, but we would need to warn everyone that the test needs urgent attention.

Reply:
I've isolated some logic here that we can quarantine without disabling the entire test. This PR performs the quarantine https://github.com/kubevirt/kubevirt/pull/5229